test/basic.html

```diff
@@ -167,19 +167,22 @@
         });
 
         test('`selectIndex()` selects an item with the given index', function() {
-          assert.equal(s2.selectedItem, undefined);
+          Polymer.Base.async(function() {
```

Comment:
These tests now wait for `iron-items-changed`, so I think after rebasing you'll find this `async` delay isn't necessary.